sentry-ruby/lib/sentry/client.rb

```diff
@@ -177,11 +177,7 @@ def send_event(event, hint = nil)
 
       event
     rescue => e
-      loggable_event_type = event_type.capitalize
-      log_error("#{loggable_event_type} sending failed", e, debug: configuration.debug)
-
-      event_info = Event.get_log_message(event.to_hash)
```

Comment:
In the past we relied on this message to look into the potentially problematic event payload. Do we have an alternative for this now?

Reply:
i want to move away from logging whole payloads for the end user, one can always log it in `before_send` if one really wants. Even on our side, we spent a lot of logging resources in python by logging huge payloads once.

Reply:
The logging here should be triggered very rarely. Most expected errors should've been captured at `HTTPTransport#send_data` (network errors...etc.) so they don't even reach here.
What's most likely captured here are errors from the `before_send*` callbacks and event/envelope serialisation errors, which with the event data will make debugging much easier for users and us.

Reply:
`send_data` re-raises here
https://github.com/getsentry/sentry-ruby/blob/c5889798c9b35c0a10f03ac071c6e289da151592/sentry-ruby/lib/sentry/transport/http_transport.rb#L65-L69

so these will end up being logged for network errors too.

Reply:
Sorry that I missed it 🤦‍♂️ 
Ok let's remove it first and find a better place to log them if we later find it necessary.

Reply:
Yes but I agree that debugging `before_send` errors is not ideal. We've also discussed this internally.